packages/lmi/tests/test_llms.py

```diff
@@ -125,6 +125,66 @@ async def test_call(self, config: dict[str, Any]) -> None:
         result = await llm.call_single(messages)
         assert isinstance(result, LLMResult)
 
+    @pytest.mark.asyncio
+    @pytest.mark.parametrize(
+        "config",
+        [
+            pytest.param(
+                {
+                    "name": CommonLLMNames.OPENAI_TEST.value,
+                    "fallbacks": [
+                        {
+                            CommonLLMNames.OPENAI_TEST.value: [
+                                CommonLLMNames.ANTHROPIC_TEST.value
+                            ]
+                        }
+                    ],
+                    "model_list": [
+                        {
+                            "model_name": CommonLLMNames.OPENAI_TEST.value,
+                            "litellm_params": {
+                                "model": CommonLLMNames.OPENAI_TEST.value,
+                                "mock_response": Exception(
+                                    "Mock exception to force fallback"
+                                ),
+                            },
+                        },
+                        {
+                            "model_name": CommonLLMNames.ANTHROPIC_TEST.value,
+                            "litellm_params": {
+                                "model": CommonLLMNames.ANTHROPIC_TEST.value,
+                                "max_tokens": 56,
+                            },
+                        },
+                    ],
+                },
+                id="multiple-models",
+            ),
+        ],
+    )
+    @pytest.mark.vcr(match_on=[*VCR_DEFAULT_MATCH_ON, "body"])
+    async def test_call_w_multiple_models(self, config: dict[str, Any]) -> None:
```

Comment:
Might be cool to generalize this test a bit to account for an actual fallback taking place